ci/values/amazon_web_services.yaml

```diff
@@ -5,3 +5,17 @@ ingress:
     enabled: true
 cert-manager:
   enabled: true
+
+#
+# For AWS ELB in L4 (TCP) mode, we need to enable some additional config
```

Comment:
Those will eventually be moved as default when `ingress.nginx.enabled` is set to `true`

Reply:
I don't think there's an easy way to override those at runtime.